.github/workflows/check_links.yml

```diff
@@ -0,0 +1,28 @@
+name: Links
+
+on:
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+    - cron: "00 18 * * *"
```

Comment:
Makes sense to add a comment to describe what this schedule is.

Reply:
I think it's arbitrary. Will change to run at midnight.